bridges/snowbridge/primitives/core/src/sparse_bitmap.rs

```diff
@@ -6,31 +6,31 @@ use sp_std::marker::PhantomData;
 /// Sparse bitmap interface.
 pub trait SparseBitmap<BitMap>
 where
-	BitMap: StorageMap<u128, u128, Query = u128>,
+	BitMap: StorageMap<u64, u128, Query = u128>,
 {
-	fn get(index: u128) -> bool;
-	fn set(index: u128);
+	fn get(index: u64) -> bool;
+	fn set(index: u64);
 }
 
 /// Sparse bitmap implementation.
```

Comment:
Documentation is very sparse (excuse the pun). Should at least describe how we are packing nonces into the `StorageMap<u64, u128>` data structure.

Reply:
Added docs in https://github.com/claravanstaden/polkadot-sdk/pull/18/commits/f88400d6025c65e1caa386b62ad5c5eb98565317.